web/client/components/widgets/enhancers/textWidget.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const {compose} = require('recompose');
+const deleteWidget = require('./deleteWidget');
+module.exports = compose(
```

Comment:
???

Reply:
Yes, compose only one enhancer is unnecessary